src/users/containers/users/users.component.ts

```diff
@@ -22,6 +22,7 @@ export class UsersComponent implements OnInit, OnDestroy {
   public currentPageNumber: number = 1;
   public pageTotalSize: number;
   public allUsersList$: Subscription;
+  public accessTypes$: Observable<string[]>;
```

Comment:
Access types should be of type strongly type. Consider writing an interface `JurisdictionAccessType`